src/platform/device.gni

```diff
@@ -19,10 +19,13 @@ declare_args() {
   # Device platform layer: cc13x2_26x2, darwin, efr32, esp32, external, freertos, linux, nrfconnect, k32w, qpg, none.
   chip_device_platform = "auto"
   chip_platform_target = ""
+  fake_platform = false
```

Comment:
Please document and add "chip_" prefix.

Actually why do we need it, since we could just set chip_device_platform = "fake" in the args?

Reply:
Did this so I could get this test running as a part of the standard ./gn_build.sh command. Otherwise it'll end up rotting and tripping up developers - if they have to build with several different --args set, it won't get built locally and will end up being a surprise annoyance at presubmit.

Reply:
Added chip_ prefix and comment.

Reply:
I don't quite follow why this is needed for gn_build.sh but we can discuss after.